### DISCUSSION TITLE
vue 3.4: issue with QInput

### DISCUSSION THREAD

**Discussion by cap1024** [quasarframework/quasar#16775] [General - Components / Directives / etc] [answered]
I've updated to vue 3.4.11 and receive following error:

`Uncaught (in promise) Maximum recursive updates exceeded in component <QInput>. This means you have a reactive effect that is mutating its own dependencies and thus recursively triggering itself. Possible sources include component template, render function, updated hook or watcher source function.`

With vue 3.3.12 works fine.

Any ideas what could be the problem?

UPDATE: with vue input works correctly.

**Comment by pdanpdan** [comment]
Can you make a reproduction in codepen?

**Reply by cap1024** [reply]
https://codepen.io/cap1024/pen/NWJbZbB

Steps to reproduce:

1. Open link - OK
2. Uncomment QInput - seems like infinite loop
3. Change vue version to 3.3.11 - OK

**Comment by pdanpdan** [comment, marked as answer]
Seems to be fixed in 3.4.13
https://codepen.io/pdanpdan/pen/eYXBwPN?editors=1010
For the moment 3.4.x are a little bit unstable, with one release per day.
So wait for it to stabilize

**Reply by nachoalcazar** [reply]
My Quasar app is broken with vue 3.4. All my pages has different components with this error "Maximum recursive updates exceeded..." and they do not render completely

With 3.4.13 OK

**Reply by Nelaka91** [reply]
I have the same issue with QHierarchy + QMarkupTable ( Vue 3.4.14)
